docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Create container based on Ubuntu-22.04 Jammy Jellyfish image
+FROM ubuntu:latest
+
+#Define arg to determine whether to build for fuse2 or fuse3
+ARG FUSE2
+
+# Create directory to hold samples
+RUN mkdir -p /usr/share/blobfuse2
+
+# Copy blobfuse2 binary to executable path
+COPY ./blobfuse2 /usr/local/bin/
+COPY ./config.yaml /usr/share/blobfuse2/
+
+# Install fuse library
+RUN \
+	apt update && \
+	apt-get install -y ca-certificates vim rsyslog
+
+RUN if [ "$FUSE2" = "TRUE" ] ; then apt-get install -y libfuse-dev fuse ; else apt-get install -y libfuse3-dev fuse3 ; fi 
```

Comment:
Are we building inside the container? or just running? if just running then we just need fuse/fuse3 right? We dont need libfuse-dev stuff

Reply:
we are just running inside the container, will check removing libfuse(3)-dev works fine or not.

Reply:
done, it works without this dependency as well.